test/Base_Tests/src/Data/Text_Spec.enso

```diff
@@ -62,7 +62,7 @@ add_specs suite_builder =
     accent_1 = '\u00E9'
     accent_2 = '\u0065\u{301}'
 
-    suite_builder.group "Text" group_builder->
+    suite_builder.group "Text.general" group_builder->
```

Comment:
```suggestion
    suite_builder.group "Text - general" group_builder->
```

nitpick - but we usually use `Type.method` to denote tests testing a specific `method` of a `Type`. Here we rather have a "section" so maybe a different symbol could be clearer?